include/emulator/errordisplay.h

```diff
@@ -0,0 +1,80 @@
+#ifndef _CODE_800633F8_H
+#define _CODE_800633F8_H
+
+#include "emulator/banner.h"
+#include "emulator/stringtable.h"
+#include "revolution/gx.h"
+#include "revolution/sc.h"
+#include "revolution/types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define FADE_TIMER_MAX 15
+
+#define FLAG_COLOR_WHITE (0 << 0)
```

Comment:
hmm, it feels weird to define the absence of a flag, especially since this implicitly means `FLAG_RESET_FADE_TIMER` is 0 too. I personally think it's fine to use 0 to mean "no flags set" but I don't feel stronly about it

Reply:
I think it's weird to not use something for the color "white" since we have something for the yellow one (and it defaults to white if the color flag isn't set)

I think it better to understand, though I can remove this from ``sStringDraw``

Reply:
I see, I'm fine with it